config/sync/field.storage.taxonomy_term.field_name.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 4348a46d-6d3a-4deb-a46d-ad3a190b1d63
+langcode: en
+status: true
+dependencies:
+  module:
+    - taxonomy
+id: taxonomy_term.field_name
+field_name: field_name
```

Comment:
Delete the name field you created. Taxonomy terms come with a name field by default.